sdk/tables/Azure.Data.Tables/tests/TableServiceClientLiveTests.cs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Core.Testing;
+using Azure.Data.Tables.Models;
+using NUnit.Framework;
+
+namespace Azure.Data.Tables.Tests
+{
+    /// <summary>
+    /// The suite of tests for the <see cref="TableServiceClient"/> class.
+    /// </summary>
+    /// <remarks>
+    /// These tests have a dependency on live Azure services and may incur costs for the associated
+    /// Azure subscription.
+    /// </remarks>
+    //[LiveOnly]
+    public class TableServiceClientLiveTests : RecordedTestBase<TablesTestEnvironment>
+    {
+        /// <summary>
+        /// The table endpoint.
+        /// </summary>
+        private readonly Uri _tableEndpoint;
+
+        public TableServiceClientLiveTests(bool isAsync) : base(isAsync /* To record tests, add this argument, RecordedTestMode.Record */)
+        {
+            _tableEndpoint = new Uri(TestEnvironment.StorageUri);
+            Sanitizer = new TablesRecordedTestSanitizer();
+            Matcher = new TablesRecordMatcher(Sanitizer);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="TableServiceClient" /> with the endpoint and API key provided via environment
+        /// variables and instruments it to make use of the Azure Core Test Framework functionalities.
+        /// </summary>
+        /// <returns>The instrumented <see cref="TableServiceClient" />.</returns>
+        private TableServiceClient CreateTableServiceClient()
+        {
+            TableServiceClient client = new TableServiceClient(_tableEndpoint,
+                                                               new TablesSharedKeyCredential(
+                                                                   TestEnvironment.AccountName,
+                                                                   string.Empty /* To record new tests, replace this value with TestEnvironment.PrimaryStorageAccountKey */
```

Comment:
Why didn't you leave the `TestEnvironment.PrimaryStorageAccountKey` here?

Reply:
It would read the variable from the recoded file https://github.com/Azure/azure-sdk-for-net/pull/11538/files#diff-1ae7d3e612a45ddcd6c3af2f35c6373dR283